.github/workflows/ci.yml

```diff
@@ -189,12 +189,18 @@ jobs:
       - name: Login to Amazon ECR
         id: login_ecr
         uses: aws-actions/amazon-ecr-login@v1
+      - run: |
+          if [[ $GITHUB_REF == $STAGING_BRANCH ]]; then
+            echo SERVERLESS_STAGE=staging >> $GITHUB_ENV;
+          elif [[ $GITHUB_REF == $PRODUCTION_BRANCH ]]; then
+            echo SERVERLESS_STAGE=production >> $GITHUB_ENV;
+          fi
```

Comment:
It looks like these file-changes were meant to fix a serverless deployment bug, and is out of scope of this PR.

As discussed in our agreement with squash-merging, doing multiple things within a PR can be dangerous with squash-merging, because fine-grained code changes can no longer be rolled back independently. Hypothetically if we needed to roll back the serverless fixes for further work, we would need to roll back your datadog integration code too (vice versa).

As such, could i trouble you to move these changes to a separate PR? thanks!

Reply:
I did this because I need the SERVERLESS_STAGE in the docker build command as a build argument.

Reply:
tough decisions to be made. sounds like we can either 

1) create a PR from a different branch that fixes SERVERLESS_STAGE. Merge it in, and then rebase this current PR on top of new `develop`
2) Allow an exception here and accept the risk of coupling two features into one PR/squashed-commit.

What are your thoughts?